ios/RNOneginiSdk.swift

```diff
@@ -177,9 +177,10 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
     func authenticateUser(_ profileId: (NSString),
                         resolver resolve: @escaping RCTPromiseResolveBlock,
                         rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
-        let profile: ONGUserProfile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.value(forKey: "profileId") as! NSObject == profileId })!;
-
-        bridgeConnector.toLoginHandler.authenticateUser(profile) {
+        let profile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.value(forKey: "profileId") as! NSObject == profileId })!
```

Comment:
Are we sure about this force unwrapping (here and in fallowing lines)?

Reply:
I'm sure because this is not my code :)